sqlalchemy_api_handler/bases/accessor.py

```diff
@@ -16,7 +20,14 @@ def get(self, path):
                     next_path = '.'.join(chunks[2:])
             else:
                 next_path = '.'.join(chunks[1:])
-            return value.get(next_path)
+            if value:
+                return value.get(next_path, with_get_path_error=with_get_path_error)
+            if with_get_path_error:
+                errors = GetPathError()
+                errors.add_error('path', f'This path {path} returns a None with entity {str(self)} at key {key}')
+                raise errors
+            return None
+
```

Comment:
Ça fonctionne bien.
Après je trouve cette fonction pas assez clair par rapport à ce qu'elle fait, je pense qu'il y a moyen de faire moins de if

Reply:
@DanielBaudry I pushed a refacto, tell me what you think thanks !